packages/react-native/src/core/storage/uploader.ts

```diff
@@ -140,14 +136,24 @@ export class IpfsUploader implements IStorageUploader<IpfsUploadBatchOptions> {
         });
 
         xhr.open("POST", `${TW_UPLOAD_SERVER_URL}/ipfs/upload`);
-        xhr.setRequestHeader(BUNDLE_ID_HEADER, appBundleId || ""); // only empty on web
+        xhr.setRequestHeader(
+          BUNDLE_ID_HEADER,
+          (globalThis as any).APP_BUNDLE_ID,
+        ); // only empty on web
         if (this.clientId) {
           xhr.setRequestHeader("x-client-id", this.clientId);
         }
 
-        xhr.setRequestHeader("x-sdk-version", version);
-        xhr.setRequestHeader("x-sdk-name", packageName);
-        xhr.setRequestHeader("x-sdk-platform", platform);
+        xhr.setRequestHeader(
+          "x-sdk-version",
+          (globalThis as any).X_SDK_VERSION,
```

Comment:
should we add a fallback here just in case ? like if for some reason i use this class without a provider in my RN app